### PR TITLE
Added power drop faq.

### PR DESCRIPTION
**Pull request by jennijuju**

### CODE REVIEW

File: docs/mine/spacerace.md [hunk 1]
```diff
@@ -165,6 +165,18 @@ If you would like to change the default gas fees to accelerate your messages, ed
   MaxWindowPoStGasFee = "50 FIL"
 ```
 
+## Why did my miner power suddenly drop?
+
+The following are some of the common reasons that can cause your power to be drop:
```
Comment (mishmosh) [inline docs/mine/spacerace.md:170]:
```suggestion
Here are some of the common reasons that your power can suddenly drop:
```

File: docs/mine/spacerace.md [hunk 2]
```diff
@@ -165,6 +165,18 @@ If you would like to change the default gas fees to accelerate your messages, ed
...
+
+1. Your PoSt messages are blocked by other messages which leads to not submitting PoSt for a deadline/window. To fix this, you can follow this [guide](https://github.com/filecoin-project/lotus/blob/master/documentation/en/mining.md#separate-address-for-windowpost-messages) and setup an separate address just for windowPoSt messages.
+
+2. Your sector data is corrupted and PoSt cannot be generated with bad underlying data. Unfortunately, we currently have no easy way to detect corrupted sectors, and the entire *runPost* fails even if only one sector is marked as faulty with the current proof APIs. However, the proofs team is working on getting our miners better APIs to mitigate this issue!
+
+3. For smaller miners, when all sectors are put into a single window, missing just one deadline/window wipes out all the power for 24 hours.
```
Comment (mishmosh) [inline docs/mine/spacerace.md:176]: Can you include an explanation of when/why this resumes?

File: docs/mine/spacerace.md [hunk 3]
```diff
@@ -165,6 +165,18 @@ If you would like to change the default gas fees to accelerate your messages, ed
...
+
+The following are some of the common reasons that can cause your power to be drop:
+
+1. Your PoSt messages are blocked by other messages which leads to not submitting PoSt for a deadline/window. To fix this, you can follow this [guide](https://github.com/filecoin-project/lotus/blob/master/documentation/en/mining.md#separate-address-for-windowpost-messages) and setup an separate address just for windowPoSt messages.
+
+2. Your sector data is corrupted and PoSt cannot be generated with bad underlying data. Unfortunately, we currently have no easy way to detect corrupted sectors, and the entire *runPost* fails even if only one sector is marked as faulty with the current proof APIs. However, the proofs team is working on getting our miners better APIs to mitigate this issue!
+
+3. For smaller miners, when all sectors are put into a single window, missing just one deadline/window wipes out all the power for 24 hours. 
+
+4. Chain/miner nodes crashed. Check `lotus sync status` in this case.
```
Comment (mishmosh) [inline docs/mine/spacerace.md:178]: Add more info about what to look for & what the user can do about it. For example, check lotus sync status, if it says X then ___, if it says Y then ___.

**Review by mishmosh — CHANGES_REQUESTED** [review]
Thanks! A few comments inline.

File: docs/mine/spacerace.md (hunk 2 above)
Comment (jennijuju) [reply docs/mine/spacerace.md:176]: Im actually not sure. @arajasek does this happen when  the miner does not have enough CPU to do all the proves?

Comment (arajasek) [reply docs/mine/spacerace.md]: @mishmosh Power will be recovered if a valid WindowPoSt is submitted 24 hours later, when the missed deadline reopens.

File: docs/mine/spacerace.md [hunk 4]
```diff
@@ -165,6 +165,18 @@ If you would like to change the default gas fees to accelerate your messages, ed
...
 
+## Why did my miner power suddenly drop?
+
+Here are some of the common reasons that your power can suddenly drop:
+
+1. Your PoSt messages are blocked by other messages which leads to not submitting PoSt for a deadline/window. To fix this, you can follow this [guide](https://github.com/filecoin-project/lotus/blob/master/documentation/en/mining.md#separate-address-for-windowpost-messages) and setup an separate address just for windowPoSt messages.
+
+2. Your sector data is corrupted and PoSt cannot be generated with bad underlying data. Unfortunately, we currently have no easy way to detect corrupted sectors, and the entire *runPost* fails even if only one sector is marked as faulty with the current proof APIs. However, the proofs team is working on getting our miners better APIs to mitigate this issue!
+
+3. For smaller miners, when all sectors are put into a single window, missing just one deadline/window wipes out all the power for 24 hours. Power will be recovered if a valid WindowPoSt is submitted 24 hours later, when the missed deadline reopens. Make sure that your miner is up and has enough CPU for PoSt.
```
Comment (mishmosh) [inline docs/mine/spacerace.md:176]:
```suggestion
3. For smaller miners, when all sectors are put into a single window, missing just one deadline/window wipes out all the power for 24 hours. Power will automatically recover if a valid WindowPoSt is submitted 24 hours later, when the missed deadline reopens. Just make sure that your miner is running and has enough CPU for PoSt.
```

File: docs/mine/spacerace.md [hunk 5]
```diff
@@ -165,6 +165,18 @@ If you would like to change the default gas fees to accelerate your messages, ed
...
+
+4. Chain/miner nodes crashed. Check `lotus sync wait` in this case, if your `Taget Height` is not equal to `Height` or status is not `Completed`, that means you are not fully synced.
```
Comment (mishmosh) [inline docs/mine/spacerace.md:178]:
```suggestion
4. Chain/miner nodes crashed. Run `lotus sync wait` and check the output. If your `Target Height` is not equal to `Height` or status is not `Completed`, that means you are not fully synced.
```

**Review by mishmosh — APPROVED** [review]
2 small sentence tweaks, otherwise LGTM!

**Pull request merged**